lib/src/pages/reg_form.dart

```diff
@@ -0,0 +1,218 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fluttertoast/fluttertoast.dart';
+import 'package:loginform_new/bloc/auth_cubit.dart';
+import 'package:loginform_new/src/pages/dashboard_page.dart';
+
+class FormPage extends StatefulWidget {
```

Comment:
Make the class name as RegistrationPage and filename as registration_page.dart .
Class name and filename should look similar